index.html

```diff
@@ -0,0 +1,452 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta name="description" content="Cusana Insights is an intelligence platform for marketers and salespeople to better understand customers and measure their KPIs on social media.">
+      <meta http-equiv="refresh" content="3600;url=https://davidobi.github.io/CusanaInsights/">
+      <meta name="keywords" content="Cusana-Insights,Sales,Marketers,Social-media,Insights,Analytics">
+      <meta name="author" content="Cusana-Insights">
+      <meta name="robots" content="index,follow">
+      <meta name="copyright" content="Cusana Insights is a startup from Meltwater Entrepreneur School of Technology in Accra Ghana which is a registered Trademark.">
+      <meta name="DC.title" content="Cusana Insights -- An intelligence platform that helps marketers & sales Experts convert customers to friends.">
+      <title>Cusana Insights </title>
+      <link href="css/bootstrap.min.css" rel="stylesheet">
+      <link href="css/animate.min.css" rel="stylesheet"> 
+      <link href="css/font-awesome.min.css" rel="stylesheet">
+      <link href="css/lightbox.css" rel="stylesheet">
+      <link href="css/main.css" rel="stylesheet">
+      <link id="css-preset" href="css/presets/preset1.css" rel="stylesheet">
+      <link href="css/responsive.css" rel="stylesheet">
+
+      <!--[if lt IE 9]>
+        <script src="js/html5shiv.js"></script>
+        <script src="js/respond.min.js"></script>
+      <![endif]-->
+      
+      <link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700' rel='stylesheet' type='text/css'>
+      <link rel="shortcut icon" href="images/drop124.png">
+      
+    </head><!--/head-->
+
+    <body>
```

Comment:
Please ensure that the body is wrapped in a container tag. and each section is wrapped in a section tag